app/check_layout.py

```diff
@@ -85,13 +85,14 @@ def check_layout(layout):
         warn("Layout doesn't specify a script.")
 
 for fname in sys.argv[1:]:
-    if fname in KNOWN_NOT_LAYOUT:
+    layout_id, _ = os.path.splitext(os.path.basename(fname))
+    if layout_id in KNOWN_NOT_LAYOUT:
         continue
     layout = parse_layout(fname)
     if layout == None:
-        print("Not a layout file: %s" % fname)
+        print("Not a layout file: res/xml/%s" % layout_id)
```

Comment:
I'd prefer not to construct fake values.
```suggestion
        print("Not a layout file: %s" % layout_id)
```

Reply:
I implemented it that way so the current check_layout.output file would be compatible. Now just the layout_id gets printed.